website/docs/r/advanced_cluster.html.markdown

```diff
@@ -287,6 +287,15 @@ replication_specs {
 
 * `disk_gb_enabled` - (Optional) Flag that indicates whether this cluster enables disk auto-scaling. This parameter defaults to true.
 * `compute_enabled` - (Optional) Flag that indicates whether instance size auto-scaling is enabled. This parameter defaults to false.
+
+~> **IMPORTANT:** If `compute_enabled` is true,  then Atlas will automatically scale up to the maximum provided and down to the minimum, if provided.
+This will cause the value of `instance_size` returned to potential be different than what is specified in the Terraform config and if one then applies a plan, not noting this, Terraform will scale the cluster back down to the original `instance_size` value.
+To prevent this a lifecycle customization should be used, i.e.:  
+`lifecycle {
+  ignore_changes = [instance_size]
+}`
+But in order to explicitly change `instance_size` comment the `lifecycle` block and run `terraform apply`. Please ensure to uncomment it to prevent any accidental changes.
+
```

Comment:
```suggestion
~> **IMPORTANT:** If `compute_enabled` is true, then Atlas will automatically scale up to the maximum provided and down to the minimum, if provided.
This will cause the value of `instance_size` returned to potential be different than what is specified in the Terraform config and if one then applies a plan, not noting this, Terraform will scale the cluster back down to the original `instance_size` value.
To prevent this a lifecycle customization should be used, i.e.:  
`lifecycle {
  ignore_changes = [instance_size]
}`
After adding the `lifecycle` block to explicitly change `instance_size` comment out the `lifecycle` block and run `terraform apply`. Please be sure to uncomment the `lifecycle` block once done to prevent any accidental changes.

```

Reply:
Just a few tweaks but looks good otherwise.